include/seqan3/range/view/take.hpp

```diff
@@ -261,17 +264,17 @@ class view_take : public ranges::view_interface<view_take<urng_t, exactly, or_th
     //!\brief The reference_type.
     using reference         = reference_t<urng_t>;
     //!\brief The const_reference type is equal to the reference type if the underlying range is const-iterable.
-    using const_reference   = std::conditional_t<const_iterable_concept<urng_t>, reference, void>;
+    using const_reference   = detail::transformation_trait_or_t<seqan3::reference<urng_t const>, void>;
```

Comment:
oh cool, that you also have some use cases for it :)

Reply:
yeah, it's quite useful!

Reply:
yeah, it's quite useful!

Reply:
echo ...  :laughing: